cli/swc_util.rs

```diff
@@ -162,3 +164,78 @@ impl AstParser {
       .unwrap_or_else(|| vec![])
   }
 }
+
+struct DependencyVisitor {
+  dependencies: Vec<String>,
+}
+
+impl Visit for DependencyVisitor {
+  fn visit_import_decl(
+    &mut self,
+    import_decl: &swc_ecma_ast::ImportDecl,
+    _parent: &dyn Node,
+  ) {
+    let src_str = import_decl.src.value.to_string();
+    self.dependencies.push(src_str);
+  }
+
+  fn visit_named_export(
+    &mut self,
+    named_export: &swc_ecma_ast::NamedExport,
+    _parent: &dyn Node,
+  ) {
+    if let Some(src) = &named_export.src {
+      let src_str = src.value.to_string();
+      self.dependencies.push(src_str);
+    }
+  }
+
+  fn visit_export_all(
+    &mut self,
+    export_all: &swc_ecma_ast::ExportAll,
+    _parent: &dyn Node,
+  ) {
+    let src_str = export_all.src.value.to_string();
+    self.dependencies.push(src_str);
+  }
+}
+
+/// Given file name and source code return vector
+/// of unresolved import specifiers.
+///
+/// Returned vector may contain duplicate entries.
+#[allow(unused)]
+pub fn analyze_dependencies(
+  file_name: &str,
```

Comment:
Is this argument necessary? What if you just used "<unknown>" instead?

Reply:
Yeah, I can remove it